### ISSUE TITLE
issue with SageMath

### ISSUE THREAD

**Issue by videlec** [MarkCBell/flipper#1]
I tried installing flipper in the docker image `sagemath/sagemath-dev:8.7` via
```
$ sage -pip install flipper
```
It proceeds fine together with its dependencies (pandas, cypari). Though when importing I obtain
```
$ sage -python
Python 2.7.15 (default, Mar 24 2019, 03:41:40) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import flipper
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/sage/sage/local/lib/python2.7/site-packages/flipper/__init__.py", line 31, in <module>
    from flipper.census import census  # noqa: F401
  File "/home/sage/sage/local/lib/python2.7/site-packages/flipper/census.py", line 9, in <module>
    DATABASES = set(os.path.splitext(os.path.basename(path))[0] for path in os.listdir(DATABASE_DIRECTORY) if os.path.splitext(path)[1] == '.csv')
OSError: [Errno 2] No such file or directory: '/home/sage/sage/local/lib/python2.7/site-packages/flipper/censuses'
```

**Comment by videlec** [comment]
Though everything is fine if flipper is installed from git.

**Comment by MarkCBell** [comment]
Thanks for letting me know about this. This was actually independent of sage and so would have occurred even if you had run ``pip install flipper``. I've updated and added MANIFEST.in to include these files and released flipper 0.13.2 where this issue should no longer occur. Could you run:
```
sage -pip install flipper
```
again and let me know if the install works please.

**Comment by videlec** [comment]
It now works perfectly! Thanks.
